.github/workflows/build-firmware.yml

```
@@ -57,8 +57,21 @@ jobs:
         uses: ./.github/actions/bundle-firmware
+  PublishSnapshot:
+    runs-on: ubuntu-latest
+    needs: [ Bundle ]
+#    if: |
+#      (github.event_name == 'push' && startsWith(github.ref, 'refs/heads/release')) ||
+#      (github.event_name == 'push' && github.ref == 'refs/heads/main')) ||
+#      (github.event_name == 'pull_request' && startsWith(github.head_ref, 'release'))
+    steps:
+      - uses: actions/checkout@v4
+      - name: "Publish Snapshot"
+        uses: ./.github/actions/publish-snapshot
```

actionlint (1.7.8)

description is required in metadata of "Bundle All" action at "/home/jailuser/git/.github/actions/publish-snapshot/action.yml"

(action)